xla/service/gpu/gpu_compiler_test.cc

```diff
@@ -681,6 +681,7 @@ ENTRY main {
     debug_options.set_xla_gpu_cublas_fallback(enable_blas_fallback);
     debug_options.set_xla_gpu_enable_triton_gemm(enable_triton);
     debug_options.set_xla_gpu_cudnn_gemm_fusion_level(0);
+    debug_options.set_xla_gpu_enable_cublaslt(false);
```

Comment:
we haven't included this test for fp8 on hipblaslt? @ScXfjiang

Reply:
I remember hipblaslt was fine when I was working on this test.

https://github.com/openxla/xla/pull/25680/files